src/commands/apply.js

```diff
@@ -0,0 +1,101 @@
+const log = require('@dhis2/cli-helpers-engine').reporter
+const fs = require('fs-extra')
+const path = require('path')
+
+const { availableCodemods: availableCodemodsLocal } = require('../utils/codemods/availableCodemods')
+const { exec } = require('./apply/exec')
+const { findAvailableCodemodsInNodeModules } = require('../utils/codemods/findAvailableCodemodsInNodeModules')
+const { getCodemodByPackageAndName } = require('../utils/codemods/getPathByPackageAndName')
+const { mergeCodemods } = require('../utils/codemods/mergeCodemods')
+
+module.exports.command = 'apply <files..>'
+module.exports.alias = 'a'
+module.exports.desc = 'Applys a codemod on a given file/directory'
+
+module.exports.builder = yargs =>
+    yargs
+        .positional('files', {
+            describe: 'Files or path the codemod should be applied to',
+            type: 'array',
+        })
+
```

Comment:
I think that `--name` should be converted to a positional argument name `<codemod>` and make it accept the structure `package:name` so it allows for example: `apply @dhis2/d2-ui:name-of-the-codemod`

And, I would like the output from `list` to be copy-pastable to the `<codemod>` positional. 

`apply name-of-codemod --package @dhis2/d2-ui` would also be useful I think.

If the `<codemod>` argument is a full path (no `:` delimiter and no extension), then it is considered an explicit codemod path, and we can drop the `--codemod-path` as well.

Reply:
Example `list` output:

```
./bin/d2-utils-codemods list
@dhis2/d2-ui:31.1.0-change-imports
@dhis2/ui:some-other-codemod

./bin/d2-utils-codemods list --package @dhis2/d2-ui 
@dhis2/d2-ui:31.1.0-change-imports
```

Example usage:

```
./bin/d2-utils-codemods apply @dhis2/d2-ui:31.1.0-change-imports

./bin/d2-utils-codemods apply 31.1.0-change-imports --package @dhis2/d2-ui 

./bin/d2-utils-codemods apply /path/to/my-codemod.js

./bin/d2-utils-codemods apply /path/to/my-codemod.js src/
```